### PR TITLE
[pt_PT] Translates capatismo, cisgenero, transgenero #279 

### LINKED ISSUE #279: 🌐 Translations pt_PT
Bem-vinda!
If you would like to help translate existing words into Portuguese, please:

1. See [selfdefined.app](selfdefined.app) for current and complete list of English-defined words.
2. Read our [Contributing Guidelines](https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md).
3. Comment here what definition(s) you would like to define.
4. Open a pull request with your translations.
5. Invite a colleague/friend to edit if you can. If you don't know anyone, we can help find an editor for you.

## To Be Defined
- [ ] 👌 [ok sign]
- [ ] ableism
- [ ] Asian-American
- [ ] East Asian
- [ ] South Asian
- [ ] Southeast Asian
- [ ] barbaric
- [ ] bierasure
- [ ] biromantic
- [ ] bisexual
- [ ] cisgender
- [ ] crazy
- [ ] derpy
- [ ] digital Blackface
- [ ] dude
- [ ] dumb
- [ ] English as Second Language
- [ ] fatphobia
- [ ] gaslighting
- [ ] gender pronouns
- [ ] Hispanic
- [ ] hysterical
- [ ] Latin American/Latino/Latina
- [ ] Latinx
- [ ] mansplain
- [ ] minoritised
- [ ] minorities
- [ ] -misia
- [ ] neopronouns
- [ ] non-binary
- [ ] Obsessive Compulsive Disorder (OCD)
- [ ] Oriental
- [ ] overrrepresented majority
- [ ] pansexual
- [ ] performative allyship
- [ ] -phobia
- [ ] polyamory
- [ ] pow-wow
- [ ] preferred pronoun
- [ ] pronouns
- [ ] r-word
- [ ] sane
- [ ] sanity check
- [ ] savage
- [ ] spaz
- [ ] suicide
- [ ] tone-deaf
- [ ] transfeminine
- [ ] transgender
- [ ] underrepresented minority
- [ ] unreal
- [ ] white feminism
- [ ] white fragility
- [ ] women and people of colour

### PR DESCRIPTION
**Pull request by goncalopereira**
My first attempt on the #279 I'll try to add more into the PR if it all looks ok.

Cheers

### CODE REVIEW

File: 11ty/definitions/pt_PT/capacitismo.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+speech: noun
+flag:
+  level: warning
+  text: content warning
+reading:
+- text: "Capacitismo (wikipedia)"
+  href: https://pt.wikipedia.org/wiki/Capacitismo
+---
+
+Um sistema que valoriza o corpo e a mente das pessoas com base em ideias construídas socialmente de normalidade, inteligência, excelência e produtividade. Essas ideias construídas estão profundamente enraizadas no anti-negritude, na eugenia, no colonialismo e no capitalismo.
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/capacitismo.md]: "no anti-negritude" should be "na anti-negritude". "Negritude" is a feminine noun in portuguese.: https://dicionario.priberam.org/negritude

File: 11ty/definitions/pt_PT/capacitismo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Capacitismo / Aptidismo
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/capacitismo.md]: I can't find any meaning for "Aptidismo".

File: 11ty/definitions/pt_PT/transgenero.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Transgênero
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/transgenero.md]: "Transgênero" is the pt_BR version. The pt_PT version should be "Transgénero".

File: 11ty/definitions/pt_PT/transgenero.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+slug: transgenero
+speech: adj
+defined: true
+excerpt: De, relacionado a, ou caracterizado por um gênero que não corresponde ao gênero ao qual foram atribuídos no nascimento.
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/transgenero.md]: Again, "gênero" should be "género".

File: 11ty/definitions/pt_PT/transgenero.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+---
+De, relacionado a, ou caracterizado por um gênero que não corresponde ao gênero ao qual foram atribuídos no nascimento.
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/transgenero.md]: And also here. "Gênero" should be "género".

File: 11ty/definitions/pt_PT/transgenero.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+O oposto de [cisgenero](/definitions/cisgenero).
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/transgenero.md]: "cisgenero" should be "cisgénero".

File: 11ty/definitions/pt_PT/cisgenero.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+title: Cisgênero
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/cisgenero.md]: Like the "Transgênero" definition, "gênero" should be "género" everytime it is referred in the definition.

So here it should be "Cisgénero".

File: 11ty/definitions/pt_PT/cisgenero.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+slug: cisgenero
+speech: adj
+defined: true
+excerpt: de, relacionado a, ou caracterizado por ser um gênero que corresponde ao gênero ao qual foram atribuídos no nascimento
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/cisgenero.md]: "Gênero" should be "género".

File: 11ty/definitions/pt_PT/cisgenero.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+De, relacionado a, ou caracterizado por ser um gênero que corresponde ao gênero ao qual foram atribuídos no nascimento.
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/cisgenero.md]: "Gênero" should be "género".

File: 11ty/definitions/pt_PT/cisgenero.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+O oposto de [transgenero] (/definitions/transgenero).
```
Comment (fabioatcosta) [inline 11ty/definitions/pt_PT/cisgenero.md]: "transgenero" should be "transgénero".

**Review by fabioatcosta — CHANGES_REQUESTED** [review]
Olá Gonçalo!

So good that you had time to do these pt_PT translations and nice to have another pt_PT speaker contributing to this project!

I made some suggestions in the "code". Hope they're helpful.

Have a great day

File: 11ty/definitions/pt_PT/capacitismo.md (hunk 1 above)
Comment (goncalopereira) [reply 11ty/definitions/pt_PT/capacitismo.md]: Well caught

**Comment by goncalopereira** [comment]
Cheers. Have been out of Portugal for quite a while and some of these are new to me in Portuguese :)

File: 11ty/definitions/pt_PT/capacitismo.md (hunk 2 above)
Comment (goncalopereira) [reply 11ty/definitions/pt_PT/capacitismo.md]: Was unsure what the correct Portuguese word is. Googling I found capacitismo and aptidismo on https://pt.qaz.wiki/wiki/Kyriarchy

**Review by goncalopereira — COMMENTED** [review]
I agree with all changes but can't merge it here only on my repo so it's updated in this pr @djquesadilla

File: 11ty/definitions/pt_PT/capacitismo.md (hunk 2 above)
Comment (fabioatcosta) [reply 11ty/definitions/pt_PT/capacitismo.md]: I see. Also found that reference there. Actually, the only one when googling. Never heard the term before and am missing references. I would stay with "Capacitismo" only.

Comment (goncalopereira) [reply 11ty/definitions/pt_PT/capacitismo.md]: Agreed

**Comment by fabioatcosta** [comment]
> I agree with all changes but can't merge it here only on my repo so it's updated in this pr @djquesadilla

I guess you can just commit and push to your fork repo. You have more info here: https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md#version-2-pull-requests. Hope it is helpful.

**Comment by goncalopereira** [comment]
Thanks @djquesadilla was a bit confused, I'm on holidays and using the GitHub mobile UI. Assumed you had a pr onto my pr not just comments.

I've updated the issues. Regards

**Review by fabioatcosta — APPROVED** [review]
Thank you, @goncalopereira! Hope to see you around in other PR's.

I prepared a cheeseboard to celebrate your first contribution: 🧀🧀🧀🥖

F *

**Comment by tatianamac** [comment]
@goncalopereira @djquesadilla Thank you both for contributing! Are these ready to merge in?

**Comment by goncalopereira** [comment]
hi @tatianamac @djquesadilla I'd like to add more but I'm currently off work for a couple weeks so feel free to merge this and I'll open a new PR ?

Regards

**Review by tatianamac — APPROVED** [review]

**Pull request merged**